docs/developers/celestia-app-wallet.md

```diff
@@ -25,7 +25,7 @@ You can pick whatever wallet name you want.
 For our example we used "validator" as the wallet name:
 
 ```sh
-celestia-appd keys add validator
+celestia-appd keys add validator --interactive
```

Comment:
[optional] we may not need to list every step explicitly but just noting that this flag will display two prompts:

```shell
$ celestia-appd keys add validator --interactive
> Enter your bip39 mnemonic, or hit enter to generate one.

> Enter your bip39 passphrase. This is combined with the mnemonic to derive the seed. Most users should just hit enter to use the default, ""

```